README.md

```diff
@@ -155,8 +155,8 @@ try project.write(path: "MyApp.xcodeproj")
 #### Adding `Home` group inside `Sources` group
 
 ```swift
-guard var sourcesGroup = project.pbxproj.objects.groups.first(where: {$0.value.name == "Sources"})?.value else { return }    
-let homeGroup = PBXGroup(reference: "xxx", children: [], sourceTree: .group, path: "Home")
+guard var sourcesGroup = project.pbxproj.objects.groups.first(where: {$0.value.name == "Sources" || $0.value.path == "Sources"})?.value else { return }    
```

Comment:
In Xcode 9 groups are created with paths by default, unless they are added via "New Group without Folder". So `name` property is not stored in project xml and then is not decoded.

But before Xcode 9 when creating folder with File-New-Group menu (instead of dragging folder into the project) only `name` property will be set and `path` will be nil.

Reply:
Thanks for the clarification. Makes sense to me! 👍